public/app/features/admin/ldap/LdapSettingsPage.tsx

```diff
@@ -392,6 +399,26 @@ export const LdapSettingsPage = () => {
           </form>
         </FormProvider>
       </Page.Contents>
+      <Modal
+        title={t('ldap-settings-page.discard-modal.title', 'Leave LDAP configuration?')}
+        isOpen={isModalOpen}
+        onDismiss={() => setIsModalOpen(false)}
+      >
```

Comment:
This doesn't seem to work as I expected. To reproduce it:
- edit some fields from LDAP form
- click on any link from the main menu on the left (for example `Dashboards`)
- the page will be redirected without showing this modal first

This works for the other providers in `Authentication`, I expected it to work the same for LDAP.

Reply:
Good call. I changed it as suggested.